docs/index.md

```diff
@@ -1175,6 +1175,15 @@ Mocha allows you to define custom reporters. For more information see the [wiki]
 
 **The HTML reporter is not intended for use on the command-line.**
 
+### Printing error messages immediately
+
+If you have long-running tests you may want to print errors immediately to the console so that you don't have to wait for the tests to finish to see why a given test failed.
```

Comment:
this is kind of a long sentence.  maybe:

> By default, error details will be output at the end of the test run.  If you have long-running tests, you may want to print errors as they occur.

Reply:
Yeah that's better